src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,118 @@
+        WebElement homeOnTheHeader = webDriver.findElement(By.xpath("//a[@href='index.html']"));
+        homeOnTheHeader.isDisplayed();
+        SOF.assertThat(homeOnTheHeader.getText()).isEqualTo("HOME");
+        WebElement contactFormOnTheHeader = webDriver.findElement(By.xpath("//a[@href='contacts.html']"));
+        contactFormOnTheHeader.isDisplayed();
+        SOF.assertThat(contactFormOnTheHeader.getText()).isEqualTo("CONTACT FORM");
+        WebElement serviceOnTheHeader = webDriver.findElement(By.cssSelector(".dropdown-toggle"));
+        serviceOnTheHeader.isDisplayed();
+        SOF.assertThat(serviceOnTheHeader.getText()).isEqualTo("SERVICE");
+        WebElement metalsAndColorsOnTheHead = webDriver.findElement(By.xpath("//a[@href='metals-colors.html']"));
+        metalsAndColorsOnTheHead.isDisplayed();
+        SOF.assertThat(metalsAndColorsOnTheHead.getText()).isEqualTo("METALS & COLORS");
+        //Assert that there are 4 images on the Index Page and they are displayed
+        WebElement icon1 = webDriver.findElement(By.className("icon-practise"));
+        icon1.isDisplayed();
+        WebElement icon2 = webDriver.findElement(By.className("icon-custom"));
+        icon2.isDisplayed();
+        WebElement icon3 = webDriver.findElement(By.className("icon-multi"));
+        icon3.isDisplayed();
+        WebElement icon4 = webDriver.findElement(By.className("icon-base"));
+        icon4.isDisplayed();
+        //Assert that there are 4 texts on the Index Page under icons and they have proper text
+        WebElement textUnder1 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To include')]"));
+        textUnder1.isDisplayed();
+        SOF.assertThat(textUnder1.getText()).isEqualTo("To include good practices\n"
+            + "and ideas from successful\n"
+            + "EPAM project");
+        WebElement textUnder2 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be flexible')]"));
+        SOF.assertThat(textUnder2.getText()).isEqualTo("To be flexible and\n"
+            + "customizable");
+        textUnder2.isDisplayed();
+        WebElement textUnder3 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'To be multiplatform')]"));
+        SOF.assertThat(textUnder3.getText()).isEqualTo("To be multiplatform");
+        textUnder3.isDisplayed();
+        WebElement textUnder4 =
+            webDriver.findElement(By.xpath("//div[@class='col-sm-3']//*[contains(text(),'Already')]"));
+        SOF.assertThat(textUnder4.getText()).isEqualTo("Already have good base\n"
+            + "(about 20 internal and\n"
+            + "some external projects),\n"
+            + "wish to get more…");
+        textUnder4.isDisplayed();
+        //Assert that there is the iframe with “Frame Button” exist
+        WebElement frameWithButton = webElement.findElement(By.xpath("//*[@id='frame']"));
```

Comment:
There's a better locator for frame (we can find the element by id). And try to use xpath only if nothing else is possible :)